kubernetes/op-scim-service.yaml

```diff
@@ -7,17 +7,17 @@ metadata:
 spec:
   type: LoadBalancer
   ports:
-  # Use this port in the standard configuration to terminate TLS at the SCIM
-  # bridge container using Let's Encrypt
-  - protocol: TCP
-    name: https
-    port: 443
-    targetPort: https
-  # Use this port when terminating TLS in front of the SCIM bridge to listen
-  # for unencrypted traffic redirected from a reverse proxy or load balancer
-  #- protocol: TCP
-  #  name: http
-  #  port: 3002
-  #  targetPort: http
+    # Use this port in the standard configuration to terminate TLS at the SCIM
+    # bridge container using Let's Encrypt
+    - protocol: TCP
+      name: https
+      port: 443
+      targetPort: https
+    # Use this port when terminating TLS in front of the SCIM bridge to listen
+    # for unencrypted traffic redirected from a reverse proxy or load balancer
+    # - protocol: TCP
+    #   name: http
+    #   port: 80
```

Comment:
is it worth making the ports configurable at default to 80?

Reply:
This is the load balancer port designation. So effectively what you are configuring here is what port to use when you access the bridge via the load balancer, i.e. `http://scim.example.com:port`.

It is already configurable in the sense that you can change `containerPort` value on the deployment:
```
            - name: http
              containerPort: 3002
```

But then you also need to ensure you set `OP_PORT` in the config map to ensure we actually listen to that port.

Reply:
I had some thoughts about setting that automatically in the Deployment spec based on the envirnoment variable, but I hadn't actually looked into it beyond that.

Perhaps something to consider for future improvement.